libs/sdk-ui-dashboard/src/presentation/topBar/menuButton/useDefaultMenuItems.tsx

```diff
@@ -182,6 +177,21 @@ export function useDefaultMenuItems(): IMenuButtonItem[] {
         isExportPdfEntitlementPresent &&
         (menuButtonItemsVisibility.pdfExportButton ?? true);
 
+    const isExportVisible =
```

Comment:
There seems to be getting lot of logic inside, maybe we should consolidate it and move it to selectors